### PR TITLE
EPAMLABSBRN-94 remake series end-point to support new naming conversion

### PR DESCRIPTION
**Pull request by ElenaSpb**
EPAMLABSBRN-94 remake series end-point to support new naming conversion

### CODE REVIEW

File: src/main/kotlin/com/epam/brn/service/SeriesService.kt [hunk 1]
```diff
@@ -11,8 +12,15 @@ class SeriesService(private val seriesRepository: SeriesRepository) {
...
+        log.debug("try to find series for groupId=$groupId")
         val series = seriesRepository.findByExerciseGroupLike(groupId)
-        log.info("try to find series for groupId=$groupId")
         return series.map { seriesEntry -> seriesEntry.toDto() }
     }
+
+    fun findSeriesForId(seriesId: Long): SeriesDto {
+        log.debug("try to find series for seriesId=$seriesId")
+        val series = seriesRepository.findById(seriesId)
+            .orElseGet { throw NoDataFoundException("no series was found for id=$seriesId") }
```
Comment (rpcep) [inline src/main/kotlin/com/epam/brn/service/SeriesService.kt]: When the series is not found, JPA throws an excaption. You are covering the original exception with this one, i would either include the original exception in this one or just let it surface and be handled by advices.

Comment (ElenaSpb) [reply src/main/kotlin/com/epam/brn/service/SeriesService.kt]: there may be no any original exception, just no series for this id

Comment (rpcep) [reply src/main/kotlin/com/epam/brn/service/SeriesService.kt]: True, but if there is we will never know which one was.

Comment (ElenaSpb) [reply src/main/kotlin/com/epam/brn/service/SeriesService.kt]: this variant works good

Comment (ElenaSpb) [reply src/main/kotlin/com/epam/brn/service/SeriesService.kt]: Ruben, I have fixed it, yes

**Pull request merged**